daemon/qrexec-client.c

```diff
@@ -174,8 +175,11 @@ int main(int argc, char **argv)
     int rc = QREXEC_EXIT_PROBLEM;
 
     if (argc < 3) {
+        if (argc == 2 && (strcmp(argv[1], "-h") == 0 || strcmp(argv[1], "--help") == 0)) {
```

Comment:
Better move the check after the getopt_long loop (or maybe even drop it completely, since there is already `optind >= argc` check below), instead of duplicating --help handling.

Reply:
OK